content/guides/building/get-source-git.md

```diff
@@ -40,141 +44,150 @@ git clone https://review.haiku-os.org/buildtools
+<img src='/files/gitProcess_0.png' alt='A diagram describing the way our Git workflow works. This picture is there for aesthetic purposes and can be skipped. After a repository is cloned locally, if you have an idea, you can convert that idea into a commit using food (a pizza is depicted in the diagram). That commit will receive some change proposals, which will result in some further changes. The circle continues until the change is ready to be included in Haiku itself.'>
 
-```sh
-git clone "ssh://<login>@git.haiku-os.org/buildtools" && scp -p <login>@git.haiku-os.org:hooks/commit-msg "buildtools/.git/hooks/"
-```
+<h4>Configuring Git locally<a name="configure_env"></a></h4>
 
-<h4>Haiku:</h4>
+<div class="alert alert-danger">
+<strong>This section is important!</strong> Failing to configure Git properly before a commit will result in an incorrect name being used in your commit, making it impossible to give you well-deserved credit for your work.</div>
 
-The `<login>@` is only needed if your currently logged in username doesn't match your `review.haiku-os.org` username.
+A global Git configuration is stored in Git configuration directory (`~/.git/` or for Haiku: `~/config/settings/git/`) and will be included in <strong>every</strong> commit.
 
 ```sh
-git clone "ssh://<login>@git.haiku-os.org/haiku"
-scp -p <login>@git.haiku-os.org:hooks/commit-msg "haiku/.git/hooks/"
+git config --global user.name "John Doe"
+git config --global user.email "john.doe@example.com"
 ```
 
-<h4>Switching from read-only to write access</h4>
-
-Want to submit your first patch? Thanks, that's great! You don't need to checkout the sources again. Instead you can update your existing copy of the source to use the commiter access. Just change the remote URL:
+Although most contributors tend to disclose their real names, we do not have
+a real name policy and plenty of contributors contribute to Haiku using a pseudonym.
+That means that it is possible for anyone to contribute.
 
-```sh
-git remote set-url origin ssh://<login>@git.haiku-os.org/haiku
-```
-
-You also should install the <a href="https://review.haiku-os.org/Documentation/user-changeid.html">Gerrit hooks to generate Change-Ids</a>:
+On macOS, you must set the following option in order to avoid problems with the
+unicode representation of filenames:
 
 ```sh
-scp -p <login>@git.haiku-os.org:hooks/commit-msg "haiku/.git/hooks/"
+git config core.precomposeunicode true
 ```
 
-<h3>Some Notes</h3>
+<h4>Setup an account on Gerrit</h4>
 
-<h4>Case Sensitive Filesystem</h4>
+<p>Log in to <a href="https://review.haiku-os.org" target="_blank">Gerrit code review</a>. For the time being, using a <a href="https://github.com">GitHub account</a> is necessary.</p>
 
 <div class="alert alert-warning">
-Haiku's source code needs to reside on a case sensitive file system.
+Make sure your SSH key is generated using <code class="code">ed25519</code>. RSA keys will <strong>not</strong> work with Gerrit.
```

Comment:
I would provide a working ssh-keygen example with the right options here. Saying "you can use ssh-keygen but make sure you get an ed25519 key" makes the tutorial not self-contained, and forces people to research ssh-keygen options elsewhere.